ui/app/AppLayouts/Wallet/SendModal.qml

```diff
@@ -56,8 +56,11 @@ ModalPopup {
             anchors.right: parent.right
             visible: !btnPreview.visible
             //% "Send"
-            label: qsTrId("command-button-send")
+            label: qsTrId("command-button-send") + " " + sendModalContent.amountInput.selectedAmount + " " + sendModalContent.amountInput.selectedAsset.symbol
```

Comment:
This is kinda quick and dirty now.. For example, when dealing with rtl languages, this entire label needs to look different. This may or may not be achievable with `qsTr/qsTrId`..

Reply:
You can easily make this work with `qsTr` (and it will be translated to `qsTrId` later). Something like this:
```qml
  qsTr("Send %1 %2").arg(sendModalContent.amountInput.selectedAmount).arg(sendModalContent.amountInput.selectedAsset.symbol)
```

Reply:
If you want more info: https://doc.qt.io/qt-5/qtquick-internationalization.html